crates/bls-crypto/src/hash_to_curve/mod.rs

```diff
@@ -59,3 +60,369 @@ pub trait HashToCurve {
         extra_data: &[u8],
     ) -> Result<Self::Output, BLSError>;
 }
+
+/// Given `n` bytes, it returns the value rounded to the nearest multiple of 256 bits (in bytes)
+/// e.g. 1. given 48 = 384 bits, it will return 64 bytes (= 512 bits)
+///      2. given 96 = 768 bits, it will return 96 bytes (no rounding needed since 768 is already a
+///         multiple of 256)
+pub fn hash_length(n: usize) -> usize {
```

Comment:
This is still defined elsewhere in try_and_increment.rs

Reply:
Thanks, removing!